develop/smart-contracts/dev-environments/hardhat.md

```diff
@@ -69,47 +62,48 @@ Before getting started, ensure you have:
     - **`test`** - contains your test files that validate contract functionality
     - **`ignition`** - deployment modules for safely deploying your contracts to various networks
 
-5. Update your Hardhat configuration file (`hardhat.config.js`) to include the plugins:
+5. Finish the setup by installing all the dependencies:
 
-    ```javascript title="hardhat.config.js" hl_lines="9-10"
-    --8<-- 'code/develop/smart-contracts/dev-environments/hardhat/hardhat.config.js:0:12'
-    --8<-- 'code/develop/smart-contracts/dev-environments/hardhat/hardhat.config.js:15:16'
-      // Additional configuration will be added later
-    --8<-- 'code/develop/smart-contracts/dev-environments/hardhat/hardhat.config.js:66:66'
+    ```bash
+    npm install
     ```
 
+    !!! note
+        This last step is needed for how the `hardhat-polkadot` plugin is set up. It will install the `@parity/hardhat-polkadot` package and all its dependencies. In the future this will be handled automatically by the plugin.
```

Comment:
```suggestion
        This last step is needed to set up the `hardhat-polkadot` plugin. It will install the `@parity/hardhat-polkadot` package and all its dependencies. In the future, the plugin will handle this automatically.
```